SU2_CFD/include/iteration_structure.hpp

```diff
@@ -1515,6 +1533,23 @@ class CDiscAdjFEAIteration : public CIteration {
                       unsigned short val_iInst,
                       unsigned short kind_recording);
 
+  /*!
+   * \brief Record a single iteration of the direct mean flow system.
```

Comment:
Thanks for keeping the doxy up to date.. double-check some of these comments though (should mean flow -> elasticity here?)